libs/ui/src/lib/global-modal/global-modal.component.scss

```
@@ -1,14 +1,16 @@
+@import "@angular/flex-layout/mq";
+
 :host {
   display: block;
 }
 
 section {
   position: relative;
 
-  .closeButton {
+  & > button {
```

the > is unnecessary SASS will achieve that for you : 
```suggestion
  button {
```

---

@phpgeekfr